distributions/nuxt-start/package.json

```diff
@@ -63,7 +63,7 @@
     "vue-client-only": "^2.0.0",
     "vue-meta": "^2.4.0",
     "vue-no-ssr": "^1.1.1",
-    "vue-router": "^3.4.6",
+    "vue-router": "3.4.8",
```

Comment:
Is there a specific reason why we pin vue-router to 3.4.8 throughout the packages?

Reply:
https://github.com/nuxt/nuxt.js/pull/8325